target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -349,6 +349,7 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "2:lan" "3:lan" "4:lan" "1:wan"
 		;;
+	netgear,r6100|\
```

Comment:
This should be dropped, we are on nand subtarget.

Reply:
Indeed, that's a stray change.